app/src/org/commcare/dalvik/dialogs/AlertDialogFactory.java

```diff
@@ -0,0 +1,143 @@
+package org.commcare.dalvik.dialogs;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.commcare.dalvik.R;
+import org.javarosa.core.services.locale.Localization;
+
+/**
+ * Created by amstone326 on 10/9/15.
+ */
+public class AlertDialogFactory {
+
+    private AlertDialog dialog;
+    private View view;
```

Comment:
These can be made `final`
Also, I think your not using our custom code formatting configuration: https://confluence.dimagi.com/display/MD/Code+Conventions